htdocs/user/bank.php

```diff
@@ -502,6 +502,25 @@
 		print '</tr>';
 	}
 
+	// Accountancy code
+	if (!empty($conf->accounting->enabled)) {
+		print '<tr><td>'.$langs->trans("AccountancyCode").'</td>';
+		print '<td>'.$object->accountancy_code.'</td></tr>';
+	}
+
+	// Employee Number
+	if (!empty($conf->accounting->enabled)) {
```

Comment:
No need to test for accounting enabled

Reply:
Hello @atm-maxime 

to see more broadly, if we accept to save the accounting data while the module is not active,
must we save the data of the other modules while they are not activated?
Should we keep a history or make sure to have the means to recreate this history?